overlord/assertstate/assertstate.go

```diff
@@ -358,5 +364,29 @@ func init() {
 
 // AutoRefreshAssertions tries to refresh all assertions
 func AutoRefreshAssertions(s *state.State, userID int) error {
-	return RefreshSnapDeclarations(s, userID)
+	db := DB(s)
+
+	fetchingSnapDeclarations := makeSnapDeclarationsFetching(s)
+
+	fetching := func(f asserts.Fetcher) error {
+		accountKeys, err := db.FindMany(asserts.AccountKeyType, nil)
+		if err != nil {
+			return err
+		}
+
+		// best effort to refresh all known account-keys
+		// TODO: refresh everything when we have a bulk endpoint
```

Comment:
It would be good to file such requests as bugs on the snapstore and replace the TODOs with an URL that allows anyone to check "is this feasible yet".